.github/workflows/pr_push.yml

```diff
@@ -103,16 +103,16 @@ jobs:
         ${{matrix.extra_build_options}}
 
     - name: Build
-      run: cmake --build ${{github.workspace}}/build --config Release -j
+      run: cmake --build ${{github.workspace}}/build --config Debug -j
```

Comment:
will these configs work with ubuntu's `extra_build_options: '-DCMAKE_BUILD_TYPE=Release'`...?

Reply:
actually, maybe here in FastBuild you could change windows from `latest` to `2019` perhsp...?

Reply:
> will these configs work with ubuntu's `extra_build_options: '-DCMAKE_BUILD_TYPE=Release'`...?

Yes, of course, it does work as you see below (All checks have passed: 66 successful checks)